packit/api.py

```diff
@@ -421,3 +424,30 @@ def status(self):
         if updates:
             logger.info("\nLatest bodhi updates:")
             logger.info(tabulate(updates, headers=["Update", "Karma", "status"]))
+
+    def run_copr_build(self, namespace, repo, clone_url, committish, chroots):
+        # get info
+        client = CoprClient.create_from_config_file()
+        owner = "dhodovsk"
```

Comment:
:D 

I'm assuming this will be a user config option.

Reply:
Ehh, leftover of early testing :D It will be "packit" ofc.

Reply:
no, please, don't hardcode it, let it be configurable via the user config